dask/dataframe/core.py

```diff
@@ -213,7 +213,7 @@ def head(self, n=5, compute=True):
         Caveat, the only checks the first n rows of the first partition.
         """
         name = 'head-%d-%s' % (n, self._name)
-        dsk = {(name, 0): (head, (self._name, 0), n)}
+        dsk = {(name, 0): (lambda x, n=n: x.head(n=n), (self._name, 0))}
```

Comment:
It's nicer to leave `n` as an explicit input to this function.  This avoids serialization issues in some cases (closures in some cases can be tricky) and makes the dask graph more informative.  A closure around a lambda is hard to inspect when debugging.